package.json

```diff
@@ -7,7 +7,7 @@
     "devproxy": "moxy -f http://api.dev.zetkin.org",
     "devserver": "next dev -p 80",
     "lint": "eslint src cypress && tsc --noEmit && echo 'Finished linting without errors!'",
-    "tdd": "concurrently 'NODE_ENV=test yarn devserver' 'yarn devproxy' 'yarn cypress open'",
+    "tdd": "concurrently \"NODE_ENV=test yarn devserver\" \"yarn devproxy\" \"yarn cypress open\"",
```

Comment:
Is this actually working now? Based on our discussion on Slack, I don't think this was a fully functional solution, but only solved part of the problem. If it's not working, it should ideally not have been part of the commit. Let's investigate together!